setup-Hhh/tt/unc-Hhh-8TeV-02.vals

```diff
@@ -1,29 +1,25 @@
-tauTau_2jet2tag			signal,SM	            	lumi_8TeV            				1.026
-tauTau_2jet2tag 			signal,SM,ZTT,TT,VV		CMS_eff_t_tautau_8TeV 				1.19
-# tauTau_2jet2tag		        signal           	        CMS_eff_t_Hhh_tautau_8TeV  		1.00
+tauTau_2jet2tag			signal,SM,TT,ZLL,VV	        lumi_8TeV            				1.026
+tauTau_2jet2tag 		signal,SM,ZTT,TT,VV		CMS_eff_t_tautau_8TeV 				1.19
 tauTau_2jet2tag			signal,ZTT  			CMS_scale_t_tautau_8TeV 			1.00        
-tauTau_2jet2tag         		ggH,ggH_SM125			CMS_scale_j_8TeV				1.05
-tauTau_2jet2tag			bbH				CMS_scale_j_8TeV				1.06
-tauTau_2jet2tag         		TT				CMS_scale_j_8TeV				1.01
-tauTau_2jet2tag         		VV				CMS_scale_j_8TeV				1.03
-tauTau_2jet2tag			signal,SM,ZJ,ZL,TT,VV 		CMS_eff_b_8TeV 					0.98
-tauTau_2jet2tag			signal,SM,ZJ,ZL,TT,VV 		CMS_fake_b_8TeV 				0.98
-tauTau_2jet2tag         		QCD				CMS_htt_QCDSyst_tautau_2jet2tag_8TeV		1.35              
-tauTau_2jet2tag			W				CMS_htt_WNorm_tautau_2jet2tag_8TeV		1.30                  
-tauTau_2jet2tag			ZTT,ZL,ZJ		        CMS_htt_zttNorm_8TeV 				1.033           
-tauTau_2jet2tag			ZTT			        CMS_htt_extrap_ztt_tautau_2jet2tag_8TeV		1.030           
+tauTau_2jet2tag         	signal,SM,TT,VV,ZTT,ZLL		CMS_scale_j_8TeV		 		1.00
+tauTau_2jet2tag			signal 				CMS_eff_b_8TeV 					1.10
+tauTau_2jet2tag			TT 				CMS_eff_b_8TeV 					1.03
+tauTau_2jet2tag			VV 				CMS_eff_b_8TeV 					1.10
+tauTau_2jet2tag			signal,SM,ZLL,TT,VV 		CMS_fake_b_8TeV 				0.98
+tauTau_2jet2tag         	QCD				CMS_htt_QCDSyst_tautau_2jet2tag_8TeV		1.40              
+tauTau_2jet2tag			ZTT,ZLL 		        CMS_htt_zttNorm_8TeV 				1.033           
+tauTau_2jet2tag			ZTT			        CMS_htt_extrap_ztt_tautau_2jet2tag_8TeV		1.06           
+tauTau_2jet2tag			ZTT				CMS_htt_ttbar_emb_8TeV				1.75
```

Comment:
The ttbar embedding uncertainty appears to get quite large in this channel - should be discussed. Need to be sure it's ok to treat as lnN and not shape. Also at the moment it will be fully correlated between categories and channels, have to think if this is what we want to do.